lang_js/parsing/parser_js.mly

```diff
@@ -871,6 +898,9 @@ decorators:
  | (* empty *) { [] }
  | decorator+   { $1 }
 
+decorators_nonempty:
```

Comment:
Why did you introduce this intermediate? why not inline and juse decorator+ directly in the other rule?